encode/avcenc.c

```diff
@@ -928,6 +962,12 @@ int avcenc_render_picture()
     if (avcenc_context.misc_parameter_roi_buf_id != VA_INVALID_ID)
         va_buffers[num_va_buffers++] =  avcenc_context.misc_parameter_roi_buf_id;
 
+    if (avcenc_context.packed_aud_header_param_buf_id != VA_INVALID_ID)
+        va_buffers[num_va_buffers++] =  avcenc_context.packed_aud_header_param_buf_id;
+
+    if (avcenc_context.packed_aud_buf_id != VA_INVALID_ID)
+        va_buffers[num_va_buffers++] =  avcenc_context.packed_aud_buf_id;
+
```

Comment:
here the order does not matter. the real submit occurred at the begin_picture().

Anyway, i put them at the beginning of the array va_buffers.